### ISSUE TITLE
Will it be possible to use tabbed pages with shell navigation?

### ISSUE THREAD

**Issue by ComptonAlvaro** [dotnet/maui#15335]
### Description

In the documentation: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/tabbedpage it is said that the tabbeded pages are not compatible with shell: "[TabbedPage](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.tabbedpage) is incompatible with .NET MAUI Shell apps, and an exception will be thrown if you attempt to use [TabbedPage](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.tabbedpage) in a Shell app.".

So the question is if it is planned it the future to support tabbed pages with shell applications.

What I want it is to can navigate from a main page to a page that is a tabbed page. I add the code of one application that expose the idea that I would like to do.

In summary, I have a main page, with no tabs, and a settings page that have two tabs, one for user settings and another tab for other settings.

This is the code: 

[NavigateToTabbedPage.zip](https://github.com/dotnet/maui/files/11601657/NavigateToTabbedPage.zip)

Thanks.

### Public API Changes

No need.

### Intended Use-Case

Used tabbed pages in shell applications.

**Comment by rachelkang** [comment]
Duplicate of #6389
